mars/tensor/expressions/base/copyto.py

```diff
@@ -106,7 +109,7 @@ def tile(cls, op):
         for out_idx in itertools.product(*(map(range, out_chunk_shape))):
             in_chunks = [t.cix[get_index(out_idx[-t.ndim:], t)] if t.ndim != 0 else t.chunks[0]
                          for t in inputs]
-            out_chunk = op.copy().reset_key().new_chunk(in_chunks, in_chunks[0].shape, index=out_idx)
+            out_chunk = op.copy().reset_key().new_chunk(in_chunks, in_chunks[1].shape, index=out_idx)
```

Comment:
Expressions unittests will check chunk shape by `calc_shape` now, so it will be covered. 

We only check tensor shape for most tests in expressions before, something may be missed, consider the expression below:

```
a = mt.random.rand(10, 20, chunk_size=5)
b = mt.ones(20, chunk_size=5)
a.dot(b).execute()
```

it would raise error before fixed due to the wrong chunk shape.

Reply:
This is a copyto op, you mean this shape error is tested by `calc_shape` then?

Reply:
Sorry to miss one line in code, I have updated the comment

Reply:
Understood

Reply:
> you mean this shape error is tested by `calc_shape` then?

yes, chunk shape is tested by `calc_shape`